packages/sign-client/test/sdk/push.spec.ts

```diff
@@ -0,0 +1,66 @@
+import axios from "axios";
+import { getSdkError } from "@walletconnect/utils";
+import {
+  initTwoClients,
+  testConnectMethod,
+  deleteClients,
+  TEST_EMIT_PARAMS,
+  TEST_RELAY_URL,
+  TEST_WEBHOOK_ENDPOINT
+} from "../shared";
+import { describe, it, expect, afterEach, beforeEach } from "vitest";
+
+describe("Push", () => {
+  let clients;
+  let sessionA;
+  beforeEach(async () => {
+    clients = await initTwoClients();
+    sessionA = await (await testConnectMethod(clients)).sessionA;
+  });
+  it("receives a prompt webhook", async () => {
+    // Register Webhook for topic
+    await axios.post(`${TEST_RELAY_URL.replace('ws', 'http')}/subscribe`, { webhook: TEST_WEBHOOK_ENDPOINT, topic: sessionA.topic }, {
+      headers: {'content-type': 'application/json'}
+    });
+
+    // Send a message which triggers the webhook to be invoked
+    const eventPayload: any = {
+      topic: sessionA.topic,
+      ...TEST_EMIT_PARAMS,
+    }
+    await clients.A.emit(eventPayload);
+    
+    // Validate webhook was called
+    const res = await axios.get(`${TEST_WEBHOOK_ENDPOINT}/${sessionA.topic}`);
```

Comment:
Might be worth having some retry here? In-case there is a delay on the Relay end?

Reply:
I first had this here but then thought YAGNI/KISS. I'll re-add if we see this fail for that reason

Reply:
:+1: Will Approve